project-koku/koku
# [COST-5161] - Drop Azure v1 support

## Jira Ticket

[COST-5161](https://issues.redhat.com/browse/COST-5161)

## Description

This change will drop Azure v1 cotst export support

We need to follow up with removing all the old code but this will prevent us processing old reports. 

## Testing

1. Checkout Branch
2. Restart Koku
3. create Azure source
4. Ingest report using V2
5. Everything should process correctly
6. ingest Azure V1 report
7. A warning message should pop up and processing will halt

## Release Notes

```markdown
* [COST-5161](https://issues.redhat.com/browse/COST-5161) Drop support for Azure v1 reports
```
